chromadb/segment/impl/metadata/sqlite.py

```diff
@@ -93,7 +93,7 @@ def max_seqid(self) -> SeqId:
 
     @trace_method("SqliteMetadataSegment.count", OpenTelemetryGranularity.ALL)
     @override
-    def count(self) -> int:
+    def count(self, request_version_context: RequestVersionContext) -> int:
```

Comment:
nit (here and elsewhere in single node code): add a comment that this param is not used?

Reply:
I put it on the type, so that I don't litter every use of the type with this. Wdyt?